ProcessResultsUtils.scala

```diff
@@ -0,0 +1,147 @@
+package controllers
+
+import org.apache.spark.sql.{DataFrame, Dataset, Row, SparkSession}
+import org.slf4j.LoggerFactory
+import play.api.Configuration
+import play.api.libs.json.{JsError, JsSuccess, JsValue, Json}
+import play.api.libs.json._ // JSON library
+import play.api.libs.json.Reads._ // Custom validation helpers
+import play.api.libs.functional.syntax._ // Combinator syntax
+
+object ProcessResultsUtils extends Serializable {
+  private val LOGGER = LoggerFactory.getLogger("ProcessResultsUtils")
+  var debugMode = false
+
+  def setDebugMode (debug: Boolean) : Unit = {
+    debugMode = debug;
+  }
+
+  def anyNotNull(row: Row): Boolean = {
+    val len = row.length
+
+    var i = 0
+    var fieldNames = row.schema.fieldNames
+    //print patientId if its the only col
+    if (len == 1 && fieldNames(0).equals(Constants.SUBJECT_ID_COL_NAME))
+      return true
+    //skip patientId
+    while (i < len) {
+      if (!fieldNames(i).equals(Constants.SUBJECT_ID_COL_NAME) && !row.isNullAt(i)) {
+        return true
+      }
+      i += 1
+    }
+    false
+  }
+
+
+  def loadTableDFFromConfig(tableFilePrefix : String, spark: SparkSession, config: Configuration,
+                            dataConfigName: String): DataFrame = {
+    LOGGER.info("PloadTableDFFromConfig")
+
+    val connInfo = config.get[String](dataConfigName)
+    if (connInfo.contains("s3a")) {
+      var dataDF: DataFrame = null
+
+      dataDF = spark.read.parquet(connInfo)
+      return dataDF
+    }
+    //Use jdbc connection:
+    val url = config.get[String]("db.mysql.url")
```

Comment:
Better add an additional check - it might be s3a, jdbc, or something unsupported, cannot it?